docs/whats-new.asciidoc

```diff
@@ -1,78 +1,113 @@
+[discrete]
+=== Detection rules enhancements
 
-*Alert enhancements*
+*Rule execution logs*
 
-On the Alerts page, the Trend and Count tables now allow you to aggregate alert data based on any aggregatable ECS field. Previously, only a handful of fields were visible.
+The new {security-guide}/alerts-ui-monitor.html#rule-execution-logs[*Rule execution logs*] tab on a rule's details page provides historical data for the rule's executions over time. Use this to understand how a particular rule is running and whether it’s creating the alerts you expect
 
 [role="screenshot"]
-image::whats-new/images/8.1/counts-table.gif[]
+image::whats-new/images/8.2/rule-exec-logs.png[]
+
+*Bulk apply a Timeline template to rules*
+
+A new {security-guide}/rules-ui-management.html#edit-rules-settings[bulk actions option] allows you to apply a Timeline template to multiple rules at once.
+
+*New Rules table filter options*
 
-The {security-guide}/alerts-ui-manage.html#view-alert-details[alert details flyout] has a new layout that highlights relevant fields. You can also now update an alert's status directly from the flyout.
+You can now {security-guide}/rules-ui-management.html#sort-filter-rules[filter the Rules table] by index pattern, MITRE ATT&CK tactic, technique (name or ID), and rule name.
+
+*Rule preview feature includes alerts table*
+
+When you create or edit a detection rule and preview it, the {security-guide}/rules-ui-create.html#preview-rules[rule preview] now includes an alerts table with the expected alerts for the rule. Use this feature to learn how noisy a rule may be before saving it. You can now also preview {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match rules].
 
 [role="screenshot"]
-image::whats-new/images/8.1/alert-details.gif[]
+image::whats-new/images/8.2/preview-rules.png[]
 
-*New Cases metrics*
+*Turn off `read` privilege warnings for detection rules*
 
-{security-guide}/cases-overview.html[Cases] now have metrics that summarize alert information and response times.
+A new Advanced Settings toggle, `securitySolution:enableCcsWarning`, allows you to {security-guide}/advanced-settings.html#enable-ccs-warning[turn off `read` privilege warnings] for detection rules using a remote cross-cluster search (CCS) index pattern.
+
+[discrete]
+=== Alert details enhancements
+
+You can now {security-guide}/alerts-run-osquery.html[run Osquery searches] from the Take action menu on the Alert details flyout.
 
 [role="screenshot"]
-image::whats-new/images/8.1/cases-kpis.png[]
+image::whats-new/images/8.2/run-osquery.png[]
+
+A new *Alert prevalence* column (1) shows the total number of alerts within the selected timeframe that have identical values. The {security-guide}/alerts-ui-manage.html#view-alert-details[Alert details flyout] also now shows linked cases (2).
```

Comment:
Kind of a minor thing, but I think it'd help to explicitly reference the image somehow so that the reference numbers are clearer. On first read, I hadn't scrolled down to the image yet, so the numbers caught me slightly by surprise.
```suggestion
In the image below, a new *Alert prevalence* column (1) shows the total number of alerts within the selected timeframe that have identical values. The {security-guide}/alerts-ui-manage.html#view-alert-details[Alert details flyout] also now shows linked cases (2).
```